server/services/parcelActionsService.js

```diff
@@ -0,0 +1,7 @@
+const actions = require('../../data/actions.json')
+
+module.exports = {
+  get: async function (parcelRef) {
+    return actions
+  }
+}
```

Comment:
Just wondering if this should call the new actions service instead of accessing the actions data directly.